package.json

```diff
@@ -7,7 +7,13 @@
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate",
-    "lint": "npx eslint --ext .vue  --ext .js ."
+    "lint": "npx eslint --ext .vue  --ext .js .",
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint"
```

Comment:
Between here and `.husky/pre-commit` what is the right place to define the actions that should happen when the hook fires? Does one call the other? Or are both needed. 

I'm thinking of a time in the future when we want to, say, also run some tests on git-commit. Would we add this here to the hooks or in `.husky/pre-commit`?

Reply:
Good question. In fact, after rereading about husky just now (https://blog.typicode.com/husky-git-hooks-javascript-config/) it turns out that this section of `package.json` is superfluous. As husky is installed via npm it redirects git to look at `.husky` for hooks.

Commands like `lint` can be defined in package.json or they can simply be put into `pre-commit` by hand or automatically like so on the command line: 

`npx husky add .husky/<hook-filename> "<command>"`

In your example it would be `npx husky add .husky/pre-commit "npm test"` and you would define `test` in the `scripts` definition of `package.json`

I will remove the `husky` section of `package.json` here, commit, and merge.

Reply:
Ah, interesting. Looks like the `package.json` way was the old way and they have since then changed to the `.husky/` way

Reply:
Edited the above comment for clarity.